pipeflow_lib.tlv

```diff
@@ -214,14 +214,8 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //            /trans
 //               $ANY     // Output transaction
 //
-m4_define_plus(['m4_stall_stage'], ['
-m4_pushdef(['m4_top'],       ['$5'])
-m4_pushdef(['m4_in_pipe'],   ['$6'])
-m4_pushdef(['m4_in_stage'],  ['$7'])
-m4_pushdef(['m4_out_pipe'],  ['$8'])
-m4_pushdef(['m4_out_stage'], ['$9'])
+\TLV m4_stall_stage(/m4_top,|m4_in_pipe,@m4_in_stage,|m4_out_pipe,@m4_out_stage)
```

Comment:
no "m4_" for macro name or args. For args, use /_top, |_in_pipe, @_in_stage, etc. '_' is the naming convention for args.

Reply:
ok  i'll edit that